src/Analysis/Codelyzer.Analysis.Build/WorkspaceBuilder.cs

```diff
@@ -56,5 +56,28 @@ public async Task<List<ProjectBuildResult>> Build()
             return ProjectResults;
         }
 
+        public async Task<List<ProjectBuildResult>> GenerateNoBuildAnalysis(Dictionary<string, List<string>> oldReferences, Dictionary<string, List<string>> references)
```

Comment:
At a high level, how do we track the issues from build using references?

Reply:
Using the same method as a normal compilation. I will add the logic for this.